tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc

```diff
@@ -1542,7 +1542,10 @@ Status MarkForCompilationPassImpl::FindCompilationCandidates() {
     // only needs to be compiled once.
     //
     // TODO(disc): support other ops having shape operands.
-    if (debug_options_.cluster_for_mlir && node->type_string() == "Fill" &&
+    auto hasShapeOperand = [](Node* node) {
+      return node->type_string() == "Fill" || node->type_string() == "Range";
```

Comment:
just curious, L1546 `node->type_string() == "Range" ` is not related to `Relu6` op?

Reply:
In my current case, the `tf.Relu6` is from a TTS model.  We need this change to support e2e lowering of this model after we  add `tf.Relu6` to the supported list of DISC in bridge side.